core/ustring.cpp

```diff
@@ -4026,6 +4026,11 @@ String rtos(double p_val) {
 	return String::num(p_val);
 }
 
+String rtos(double p_val, int p_decimals) {
```

Comment:
I think you could use `String::num()` directly instead of adding an overload. `rtos()` is for convenience when debugging, but if you need actual String formatting, it's better to use String methods directly.